roles/dhcp-server/templates/pxe.conf.j2

```diff
@@ -0,0 +1,24 @@
+# /etc/dnsmasq.d/pxe.conf
+
+# https://wiki.fogproject.org/wiki/index.php?title=BIOS_and_UEFI_Co-Existence#Using_ProxyDHCP_.28dnsmasq.29
+# http://forum.ipxe.org/showthread.php?tid=6077
+
+# Tag requests from ipxe client
+dhcp-match=set:ipxe,175 # iPXE sends a 175 option.
+
+# tag/respond to legacy bios
+dhcp-match=set:bios,60,PXEClient:Arch:00000
+dhcp-boot=tag:bios,ipxe/undionly.kpxe #,dc10b,192.168.1.4
```

Comment:
What's with all the dc10b comments?

Reply:
example on how to use a different box for tftp.  
perfect for: leave the working pxe stuff in place (local dhcp box) use a test box for tftp/http/etc.

it doesn't move the dhcp stuff, but not all hacking requires touching the dhcp stuff. 

ps, I am not really sure why it needs a hostname and IP. 

pps dc10b is paying homage to the machine I won at DebCof 10 - I liked it so much I bought a 2nd one when the first one's fan started making a racket.

Reply:
Sorry, this was a somewhat rhetorical question, I was reminding you to clean them up.

I don't think we need examples in here, the place you configure this is in the variables, not by modifying the template.

Reply:
I'm not sure I agree with adding formal support for R&D.  which seems at odds with the r&d role and a bunch of other stuff that is making me question my sanity. 

if I promise to clean this up some day can we leave in here for now?